tableconvert/src/tableconvert.h

```diff
@@ -71,5 +71,7 @@ extern TableConvertRule tablerules[];
 extern void cb_table_convert(G_GNUC_UNUSED GtkMenuItem *menuitem, G_GNUC_UNUSED gpointer gdata);
 extern void cb_table_convert_type(G_GNUC_UNUSED GtkMenuItem *menuitem, gpointer gdata);
 extern void convert_to_table(gboolean header, gint file_type);
+extern void cb_table_convert_change_document(G_GNUC_UNUSED GtkMenuItem *menuitem, gpointer gdata);
+extern void set_activate_state();
```

Comment:
I suggest to move this two function declarations above to the beginning of ```tableconvert.c```. And I would remove the not required ```extern```.